Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -515,7 +514,23 @@ private void OnWorkspaceFolderWatched(object sender, EventArgs e) {
 
         private void WatchedFilesRegistered(object sender, DidChangeWatchedFilesRegistrationOptions e) {
             // Add the file globs to our listener. It will listen to the globs
+            UpdateInterpreterExcludes();
```

Comment:
on startup update excludes so that ptvs doesnt' send fs events under the interpreter folder